LibSWBF2/Wrappers/GlobalLighting.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+
+
+#include "Chunks/LVL/lght/lght.h"
+#include "InternalHelpers.h"
+
+#include "Types/Colorf.h"
+#include "Types/Vector4.h"
+#include "Types/Vector3.h"
+
+
+namespace LibSWBF2::Wrappers
+{
+
+using namespace LibSWBF2::Chunks::LVL::lght;
+using namespace LibSWBF2::Types;
+
+struct GlobalLightingConfig {
+
+public:
+
+	bool GetTopColor(Vector3& colorOut);
+	bool GetBottomColor(Vector3& colorOut);
+
+	bool GetLight1(String& light1NameOut);
+	bool GetLight2(String& light2NameOut);
+
+	GlobalLightingConfig(SCOP_LGHT* body);
+	GlobalLightingConfig() = default;
+
+	String ToString();
+	static bool FromChunk(SCOP_LGHT* body, GlobalLightingConfig& out);
+
+
+private:
+	SCOP_LGHT* p_FieldsChunk;
+};
+
+}
```

Comment:
Maybe this wrapper is overkill,  I added it to encapsulate the single (not per layer, there is always one per lvl file) ```SCOP``` holding global lighting data for the level file.

Reply:
Not overkill at all, exactly what we need aswell. Good work!